clash-lib/clash-lib.cabal

```diff
@@ -297,6 +302,20 @@ Library
   if flag(debug)
     cpp-options:      -DDEBUG
 
+executable v16-upgrade-primitives
```

Comment:
Hmm, Cabal would only install the binaries if you explicitly do `cabal install clash-lib`, right? I don't imagine people would do that. I still have to work out the exact instructions, but I think people would run it with  something like `cabal run clash-lib:v16-upgrade-primitives`. This would make the additional `clash-` redundant.

Reply:
:+1: 
I hadn't thought of the `run clash-lib:`